data/companies.yml

```diff
@@ -95,7 +95,7 @@ providers:
     logo: "/logos/microsoft-azure.svg"
     url: "https://learn.microsoft.com/en-us/azure/aks/istio-about"
   - name: "IMESH Platform"
-    logo: "/logos/imesh.svg"
+    logo: "/logos/imesh-logo.svg"
```

Comment:
can we just use the old name itself? why do we need a -logo in the name?

Reply:
> can we just use the old name itself? why do we need a -logo in the name?

Yes, Now I have renamed imesh-logo.svg to imesh.svg